blogposts/2020/optimizing-a-code-intelligence-indexer.md

```diff
@@ -0,0 +1,196 @@
+---
+title: "Optimizing a code intelligence indexer"
+author: Eric Fritz
+authorUrl: https://eric-fritz.com
+publishDate: 2020-08-17T08:00-07:00
+tags: [blog]
+slug: optimizing-a-code-intel-indexer
+heroImage: https://sourcegraphstatic.com/blog/lsif-go/lsif-go-improvements.png
+published: true
+---
+
+We've finally tagged the [one-point-oh release](https://github.com/sourcegraph/lsif-go/releases/tag/v1.0.0) of [lsif-go](https://github.com/sourcegraph/lsif-go) boasting impressive speed improvements that brings intelligence to your Go code faster than ever before.
+
+The code intelligence is hard at work trying to bring the _time to intelligence_ - the time between a commit and useful navigation of that code in your Sourcegraph instance - as close to zero as possible. We want your code to always have up-to-date precise code intelligence on the newest commits to help aid you when browsing and reviewing unfamiliar code. Indexing speed is a _huge_ part of this, and we've made a great leap here.
+
+As an illustrative example, checkout the visual difference between indexing our [main repository](https://github.com/sourcegraph/sourcegraph) with lsif-go v0.9 (above) and indexing it with lsif-go v1.0 (below).
+
+[![asciicast](https://asciinema.org/a/pKTby2O4N1KC9RqXBkOO7JgBw.svg)](https://asciinema.org/a/pKTby2O4N1KC9RqXBkOO7JgBw)
```

Comment:
It would be neat to be able to play these side-by-side. Not a blocker.

Reply:
I had tried doing this but wasn't sure how - they also don't autoplay. I originally had gif recordings but am not gifted though to figure out how to combine them without paying Adobe a subscription to the creative suite. If anyone has suggestions on _how_ to do this I'll spend the time.